### PR TITLE
[IMP] accounting: OSS report

### PR DESCRIPTION
**Pull request by toaa-odoo**
Added a section on where to find OSS reports and how to export them.

16.0 --> Change screenshot of report menu

Forward to master after.

task-3305037

### CODE REVIEW

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
Hey @toaa-odoo , as usual, a great work. Here are my suggestions, mostly to avoid "repetitions in ideas" - we can have a call to discuss it if you want to :)

File: content/applications/finance/accounting/taxes/eu_distance_selling.rst [hunk 1]
```diff
@@ -2,68 +2,79 @@
...
 
-Refresh tax mapping
--------------------
+Reports
+=======
 
-Whenever you add or modify taxes, you can update automatically your fiscal positions.
+To generate **OSS sales** or **OSS imports reports** and submit them onto the OSS portal, go to
+:menuselection:`Accounting --> Reporting --> Tax Report`, click :guilabel:`Tax Report: Global
+Summary`, and select either :guilabel:`OSS Sales` or :guilabel:`OSS Imports`. Once selected, click
```
[the remark this answers is not in the document]
Comment (toaa-odoo) [reply content/applications/finance/accounting/taxes/eu_distance_selling.rst:54]: Thanks for the notification!

**Review by LoredanaLrpz — APPROVED** [review]
Hey @toaa-odoo , I just have one last suggestion, but I'm approving this PR
Cheers!

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hi @toaa-odoo and @LoredanaLrpz 

In addition to my comments, you can also remove the centered alignment of the images if you'd like.

Cheers :)

**Comment by jcs-odoo** [comment]
@robodoo r+

**Pull request closed without merging**